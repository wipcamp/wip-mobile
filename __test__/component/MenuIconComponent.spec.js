/**
 * @jest-environment node
 */

import React from 'react'
import renderer from 'react-test-renderer'

import MenuIcon from '../../components/MenuIconComponent'

it('rendering', () => {
    const tree = renderer
        .create(
            <MenuIcon
                function={jest.fn()}
                icon={{}}
                text="test"
            />
        )
        .toJSON()
    expect(tree).toMatchSnapshot()
})

it('rendering two', () => {
    const tree = renderer
        .create(
            <MenuIcon
                function={jest.fn()}
                icon={{}}
                text="test"
                two
            />
        )
        .toJSON()
    expect(tree).toMatchSnapshot()
})