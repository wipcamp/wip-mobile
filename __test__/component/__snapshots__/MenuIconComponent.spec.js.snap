// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  collapsable={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFF",
      "borderBottomWidth": 0.5,
      "borderColor": "#C8C8C8",
      "borderRadius": 4,
      "flex": 0.45,
      "flexDirection": "column",
      "justifyContent": "center",
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Image
    source={Object {}}
    style={
      Array [
        Object {
          "marginBottom": 5,
        },
        Object {
          "height": 19,
          "resizeMode": "contain",
        },
      ]
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontFamily": "Kanit",
        },
        Object {
          "paddingTop": 5,
        },
        Object {
          "fontSize": 16,
        },
      ]
    }
  >
    test
  </Text>
</View>
`;

exports[`rendering two 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  collapsable={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFF",
      "borderBottomWidth": 0.5,
      "borderColor": "#C8C8C8",
      "borderRadius": 4,
      "flex": 0.9,
      "flexDirection": "row",
      "justifyContent": "space-around",
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Image
    source={Object {}}
    style={
      Array [
        Object {
          "left": 9.5,
        },
        Object {
          "height": 25,
          "width": 25,
        },
      ]
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontFamily": "Kanit",
        },
        Object {
          "right": 9.5,
        },
        Object {
          "fontSize": 25,
        },
      ]
    }
  >
    test
  </Text>
</View>
`;
